features/step_definitions/timehelpers_steps.rb

```diff
@@ -33,18 +33,19 @@ def access_user_sessions
   access_user_sessions
 end
 
-When /^(?:the )?time flies to (.*)$/ do |date|
+When /^(?:the )?time flies to ([^\(]*)( \(without jobs\))?$/ do |date, skip_jobs|
   date = date.gsub(Regexp.union(%w[of st nd rd]), '')
-  time_machine(Time.zone.parse(date))
+  time_machine_opts = skip_jobs ? { skip_jobs: true } : nil
+  time_machine(Time.zone.parse(date), time_machine_opts)
   step %(the date should be #{date})
   access_user_sessions
 end
 
 # Suffix 'on 5th July 2009'
 #
-Then /^(.+) on (\d+(?:th|st|nd|rd) \S* \d{4}(?: .*)?)$/ do |original, date|
+Then /^(.+) on (\d+(?:th|st|nd|rd) \S* \d{4}(?: [^\(]*)?)( \(without jobs\))?$/ do |original, date, skip_jobs|
   # this ensures billing actions are run
```

Comment:
What are "billing actions"?

This is to avoid all daily jobs running for every day from the initial date to the target date (a defined in `time_machine` method)

Reply:
I don't know, it's what the comment says. But I assume billing actions are some kind of "scheduled job" and this step skips them so I figured this comment is no longer accurate because the step skips jobs, including billing ones. Does this make sense?

Reply:
Aaaah, OK, thanks for the clarification, I didn't see that comment :eyes: :sweat_smile: 

Yes, this comment is confusing, I will remove it and will add another one clarifying the behavior of the step.